pkg/bindings/generate/generate.go

```diff
@@ -4,31 +4,23 @@ import (
 	"context"
 	"errors"
 	"net/http"
-	"net/url"
-	"strconv"
 
 	"github.com/containers/podman/v2/pkg/bindings"
 	"github.com/containers/podman/v2/pkg/domain/entities"
 )
 
-func Systemd(ctx context.Context, nameOrID string, options entities.GenerateSystemdOptions) (*entities.GenerateSystemdReport, error) {
+func Systemd(ctx context.Context, nameOrID string, options *SystemdOptions) (*entities.GenerateSystemdReport, error) {
```

Comment:
`entities.GenerateSystemdReport` as a return value is escaping the bindings.  Are there plans to do the same dance for return values?

Reply:
i cannot see this getting done in this round.